### PR TITLE
added script to grab environment variables from azure tags

### PR DESCRIPTION
**Pull request by zmmille2**

### CODE REVIEW

File: scripts/azure_tags_env_variables.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from subprocess import Popen, PIPE
+import json
+
+raw_response = Popen(["curl", "-H", "Metadata:true", "http://169.254.169.254/metadata/instance?api-version=2017-08-01"], stdout=PIPE)
+res = raw_response.stdout.read()
+
+res_json = json.loads(res)
+res_tags = res_json['compute']['tags']
+
+file_obj = open("AzureTagsEnvVariables", "w")
```
Comment (aluong) [inline scripts/azure_tags_env_variables.py]: You can do: 

open("AzureTagsEnvVariables", "w") as file_obj:

File: scripts/azure_tags_env_variables.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from subprocess import Popen, PIPE
```
Comment (aluong) [inline scripts/azure_tags_env_variables.py:2]: Can we add a comment describing what this does?

File: scripts/azure_tags_env_variables.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+raw_response = Popen(["curl", "-H", "Metadata:true", "http://169.254.169.254/metadata/instance?api-version=2017-08-01"], stdout=PIPE)
+res = raw_response.stdout.read()
+
+res_json = json.loads(res)
+res_tags = res_json['compute']['tags']
+
+file_obj = open("AzureTagsEnvVariables", "w")
+for pair in res_tags.split(';'):
+  pair = pair.encode('ascii', 'ignore').upper()
+  key_value = pair.split(':')
```
Comment (aluong) [inline scripts/azure_tags_env_variables.py]: Maybe print out the tags for debugging via the logs?

**Review by aluong — APPROVED** [review]

File: scripts/azure_tags_env_variables.py (hunk 1 above)
Comment (zmmille2) [reply scripts/azure_tags_env_variables.py]: makes sense.  I've put this in a with block to make things cleaner.

File: scripts/azure_tags_env_variables.py (hunk 2 above)
Comment (zmmille2) [reply scripts/azure_tags_env_variables.py:2]: yep!

File: scripts/azure_tags_env_variables.py (hunk 3 above)
Comment (zmmille2) [reply scripts/azure_tags_env_variables.py]: Not sure if this is what you mean, let me know if you want me to look into importing some kind of logger

**Pull request merged**